code/game/machinery/hologram.dm

```diff
@@ -47,17 +69,30 @@ GLOBAL_LIST_EMPTY(holopads)
 	max_integrity = 300
 	armor = list(melee = 50, bullet = 20, laser = 20, energy = 20, bomb = 0, rad = 0, fire = 50, acid = 0)
 
-	var/list/masters = list()//List of living mobs that use the holopad
-	var/list/holorays = list()//Holoray-mob link.
-	var/last_request = 0 //to prevent request spam. ~Carn
-	var/holo_range = 5 // Change to change how far the AI can move away from the holopad before deactivating.
+	/// List of living mobs currently using the holopad
+	var/list/masters = list()
+	/// Holoray-mob link.
+	var/list/holorays = list()
+	/// Last request time, to prevent request spam. ~Carn
+	var/last_request = 0 
+	/// Change to change how far the AI can move away from the holopad before deactivating.
+	/// The range, in tiles, that a holopad can project a hologram.
```

Comment:
I guess the above comment can be deleted? The one you added explains it well enough.